atlp-rwanda/Avengers-Phantom-BE
# Setting up a docker

### what does this PR do?
- Setting up a docker
### Task description
- Added a docker file consisting of configurations to build project image
- added docker.compose.yml to run both project and database containers
### How to test it manually
- Firstly you need to have docker installed
- open your terminal and start building both postgres and project image
`docker build -t avengersphantom-be .` to build project image and `docker run --name some-postgres -e POSTGRES_PASSWORD=mysecretpassword -d postgres` to download and run postgres image
- and `docker-compose up -d ` to run both image in the same container then  run `docker exec <server container id> npm run migrate `
- use `docker ps` command to see running containers information
- you can see that i have them running on my laptop
![Capture](https://user-images.githubusercontent.com/50317787/172047157-0eaa97b4-8b06-4f23-b829-6ad5b6178b99.PNG)
s
